packages/node_modules/@webex/webex-core/test/unit/spec/services/service-catalogs.js

```diff
@@ -406,13 +449,57 @@ describe('webex-core', () => {
         ));
       });
 
-      it('should return no hosts with an invalid service', () => {
+      it('should return an empty array when given an invalid service', () => {
         filteredHosts = serviceCatalogs.filterService('invalid');
 
         assert.equal(filteredHosts.length, 0);
       });
     });
 
+    describe('#filterUrl()', () => {
+      let filteredHosts;
+      let hostA;
```

Comment:
Let's do `filteredHostA` and `filteredHostB` here as well.

Reply:
Will make this change.